2024-05-14 15:23:27,980 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:23:29,610 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:23:32,425 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:23:32,425 INFO: Processing URL: https://medium.com/@codelancingg/how-to-downgrade-python-version-fb7b9087e776 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:248]
2024-05-14 15:23:38,354 ERROR: Error occurred: 'NoneType' object is not iterable [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:330]
2024-05-14 15:23:38,355 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:66]
2024-05-14 15:24:32,084 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:24:51,249 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:27:02,796 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:27:42,975 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:28:10,139 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:28:23,657 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:55]
2024-05-14 15:28:26,477 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:28:26,479 INFO: Processing URL: https://medium.com/@codelancingg/how-to-downgrade-python-version-fb7b9087e776 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:249]
2024-05-14 15:28:39,429 ERROR: Error occurred: 'NoneType' object is not iterable [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:333]
2024-05-14 15:28:39,430 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:66]
2024-05-14 15:30:39,355 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:30:39,356 INFO: Processing URL: https://medium.com/@codelancingg/how-to-downgrade-python-version-fb7b9087e776 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:249]
2024-05-14 15:30:54,652 INFO: Outgoing response: 200 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:66]
2024-05-14 15:33:06,333 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:33:06,334 INFO: Processing URL: https://medium.com/@codelancingg/how-to-downgrade-python-version-fb7b9087e776 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:249]
2024-05-14 15:33:18,754 INFO: Outgoing response: 200 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:66]
2024-05-14 15:35:19,769 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:35:19,769 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:249]
2024-05-14 15:35:57,194 ERROR: Error occurred: 'NoneType' object is not iterable [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:333]
2024-05-14 15:35:57,197 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:66]
2024-05-14 15:38:32,406 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:41:24,089 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:41:32,976 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 15:41:32,978 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:242]
2024-05-14 15:41:51,131 ERROR: Error occurred: Error code: 400 - {'error': {'message': "Invalid 'tools': empty array. Expected an array with minimum length 1, but got an empty array instead.", 'type': 'invalid_request_error', 'param': 'tools', 'code': 'empty_array'}} [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:326]
2024-05-14 15:41:51,133 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:69]
2024-05-14 15:42:42,105 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 15:42:42,106 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:242]
2024-05-14 15:42:44,274 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 15:42:44,275 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:242]
2024-05-14 15:42:58,227 ERROR: Error occurred: Error code: 400 - {'error': {'message': "Invalid 'tools': empty array. Expected an array with minimum length 1, but got an empty array instead.", 'type': 'invalid_request_error', 'param': 'tools', 'code': 'empty_array'}} [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:326]
2024-05-14 15:42:58,229 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:69]
2024-05-14 15:42:58,344 ERROR: Error occurred: Error code: 400 - {'error': {'message': "Invalid 'tools': empty array. Expected an array with minimum length 1, but got an empty array instead.", 'type': 'invalid_request_error', 'param': 'tools', 'code': 'empty_array'}} [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:326]
2024-05-14 15:42:58,347 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:69]
2024-05-14 15:43:44,089 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:43:45,477 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 15:43:45,478 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:242]
2024-05-14 15:44:40,019 ERROR: Error occurred: Error code: 400 - {'error': {'message': "Invalid 'tools': empty array. Expected an array with minimum length 1, but got an empty array instead.", 'type': 'invalid_request_error', 'param': 'tools', 'code': 'empty_array'}} [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:326]
2024-05-14 15:44:40,023 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:69]
2024-05-14 15:46:37,382 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:46:40,774 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:46:41,838 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:65]
2024-05-14 15:46:41,839 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:251]
2024-05-14 15:46:52,620 INFO: quizzes 5 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:162]
2024-05-14 15:46:53,190 ERROR: Error occurred: name 'concurrent' is not defined [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:335]
2024-05-14 15:46:53,193 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:71]
2024-05-14 15:48:11,622 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:48:19,217 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:48:22,402 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:60]
2024-05-14 15:55:51,596 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:57:37,174 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:58:26,466 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 15:59:54,842 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 16:00:10,125 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 16:00:10,126 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:268]
2024-05-14 16:00:14,219 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 16:00:39,792 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:63]
2024-05-14 16:00:39,793 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:268]
2024-05-14 16:01:41,103 ERROR: Error occurred: Max retries exceeded. Unable to get a valid response. [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:352]
2024-05-14 16:01:41,105 INFO: Outgoing response: 500 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:69]
2024-05-14 16:10:28,760 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 16:11:10,995 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:58]
2024-05-14 16:11:26,229 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:59]
2024-05-14 16:12:04,515 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:59]
2024-05-14 16:12:26,999 INFO: Application startup [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:59]
2024-05-14 16:12:37,719 INFO: Incoming request: POST http://localhost:5111/fetch [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:64]
2024-05-14 16:12:37,720 INFO: Processing URL: https://en.wikipedia.org/wiki/Empire_of_Japan [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:288]
2024-05-14 16:12:49,059 INFO: quizzes 5 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:173]
2024-05-14 16:12:59,021 INFO: Quiz [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:184]
2024-05-14 16:12:59,021 INFO: Quiz [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:184]
2024-05-14 16:12:59,022 INFO: Quiz [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:184]
2024-05-14 16:12:59,023 INFO: Quiz [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:184]
2024-05-14 16:12:59,023 INFO: Quiz [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:184]
2024-05-14 16:13:04,898 INFO: quizzes 5 [in D:\KCM\Tasks\Lurny\Quiz-app\new-app.py:173]
